docs/infrastructure.md

```diff
@@ -128,6 +128,7 @@ When a pull request is opened to `main` only the Test stage runs.
 
     ```bash
     cf create-service postgres tiny-unencrypted-13 dluhc-core-staging-postgres
+    cf create-service aws-s3-bucket default dluhc-core-staging-csv-bucket
```

Comment:
Looks like the redis setup is missing from these docs as well actually. Could you add it?

Reply:
Sure. I'll add it in a follow up PR.